packages/core/types/index.js

```diff
@@ -960,17 +995,33 @@ export interface BundleGraph<TBundle: Bundle> {
     | {|type: 'bundle_group', value: BundleGroup|}
     | {|type: 'asset', value: Asset|}
   );
+  /**
+   * @returns whether the dependency is deferred or not. Dependencies are deferred
+   *         if
```

Comment:
```suggestion
   * A dependency caused by a re-export can be deferred (skipped) if there is no
   * incoming dependency that uses the re-exported value.
   * @returns Whether the dependency was deferred.
```
Does that make sense? 😄

Reply:
yeah, thanks 😄